install/scripts/islandora-karaf-components.sh

```diff
@@ -9,3 +9,12 @@ fi
 
 echo "Installing Islandora Sync Gateway"
 $KARAF_CLIENT -f $KARAF_CONFIGS/islandora_sync_gateway.script
+sleep 10
+echo "Installing Islandora Basic Image Service"
+$KARAF_CLIENT -f $KARAF_CONFIGS/islandora_basic_image.script
+sleep 10
+echo "Installing Islandora Collection Service"
+$KARAF_CLIENT -f $KARAF_CONFIGS/islandora_collection.script
+sleep 10
```

Comment:
I just realized that we don't need or want to load the old camel Basic Image service and Collection Service. We should only load the Sync Gateway and Triplestore Indexer.